components/order_desk/order_desk.app.mjs

```diff
@@ -0,0 +1,280 @@
+import { axios } from "@pipedream/platform";
+import options from "./common/options.mjs";
+
+export default {
+  type: "app",
+  app: "order_desk",
+  propDefinitions: {
+    order_items: {
+      type: "object",
+      label: "Order Items",
+      description: "Order Items Object can have the following properties: `id, name, price, quantity, weight, code, delivery_type, category_code, variation_list, metadata`. Please check the documentation in the description for more information.",
+    },
```

Comment:
Humm, it is not possible, since `order_item` is directly linked with `order`.
One `order` has `N` `order_item` and 
One `order_item` belongs to an `order` 
Also, we can't have an array of objects in Pipedream right now. So even for updates, it wouldn't be possible to fetch all `order_items`.